benchmark/benchmark.py

```diff
@@ -103,9 +103,32 @@ def get_benchmarks():
 def main(hostpython):
     with conftest.spawn_web_server() as (hostname, port, log_path):
         results = {}
-        for k, v in get_benchmarks():
-            print(k)
-            results[k] = run_all(hostpython, port, v)
+        selenium_backends = {}
+
+        b = {'native': float('NaN')}
+        for name, cls in [('firefox', conftest.FirefoxWrapper),
+                          ('chrome', conftest.ChromeWrapper)]:
+            t0 = time()
+            selenium_backends[name] = cls(port)
+            b[name] = time() - t0
+        results['selenium init'] = b
+        print_entry("selenium init", b)
+
+        # load packages
+        for package_name in ["numpy"]:
```

Comment:
Should we add `scipy` here, too?  That would be another good one to know.

Reply:
Also, I feel like these benchmarks should start from a clean selenium instance to avoid possibly measuring the effects of caching